components/giftbox.vue

```diff
@@ -50,10 +50,16 @@
           <!-- eslint-disable-next-line vue/no-v-html -->
           <div class="feature-content" v-html="selectedFeature.body" />
           <div class="flex-right">
-            <b-button class="btn-call-to-action" variant="outline-primary">
-              {{ selectedFeature.callToActionLabel }}
-            </b-button>
-          </div>          
+            <a
+              v-if="selectedFeature.callToAction"
+              :href="selectedFeature.callToAction"
+              target="_blank"
+            >
+              <b-button class="btn-call-to-action" squared variant="primary">
```

Comment:
If all our buttons are supposed to be `squared`, maybe we should change it in the bootstrap variables _or_ create our own button component that uses `b-button` and just proxies most props, but also adds `squared` so we don't always have to repeat that.

Reply:
True.
Also why is there a button inside an `<a>` tag?

Reply:
@brampauwelyn @bertramakers I was wondering if this isn't best done in a separate PR, because it is not specific to the giftbox-feature

Reply:
I'd rather do that as soon as possible, so we don't have to go back and remove the `squared` prop later when we do override the bootstrap radius variable, or change existing usages of `b-button` to our own button component once we have one if we go that route instead.